src/app/(dashboard)/teams/new/page.tsx

```diff
@@ -74,7 +74,7 @@ export default function CreateTeamPage() {
         <CreateTeamStepper currentStep={currentStep}></CreateTeamStepper>
         {isLoading ? (
           <div className="pt-36 flex justify-center">
-            <LoadingCircle></LoadingCircle>
+            <Circles color="black" />
```

Comment:
why did you removed the LoadingCircle that uses the correct color depending on the theme to add the one that only works on light mode?